### PR TITLE
Dotnet Skills auth

### PR DESCRIPTION
**Pull request by EricDahlvang**

### CODE REVIEW

File: CSharp/Library/Microsoft.Bot.Connector.NetCore/Microsoft.Bot.Connector.NetCore.nuget.props [hunk 1]
```diff
@@ -3,11 +3,11 @@
   <PropertyGroup Condition=" '$(ExcludeRestorePackageImports)' != 'true' ">
     <RestoreSuccess Condition=" '$(RestoreSuccess)' == '' ">True</RestoreSuccess>
     <RestoreTool Condition=" '$(RestoreTool)' == '' ">NuGet</RestoreTool>
-    <ProjectAssetsFile Condition=" '$(ProjectAssetsFile)' == '' ">C:\source\github\BotBuilder-V3\CSharp\Library\Microsoft.Bot.Connector.NetCore\project.lock.json</ProjectAssetsFile>
+    <ProjectAssetsFile Condition=" '$(ProjectAssetsFile)' == '' ">$(MSBuildThisFileDirectory)project.lock.json</ProjectAssetsFile>
     <NuGetPackageRoot Condition=" '$(NuGetPackageRoot)' == '' ">$(UserProfile)\.nuget\packages\</NuGetPackageRoot>
-    <NuGetPackageFolders Condition=" '$(NuGetPackageFolders)' == '' ">$(UserProfile)\.nuget\packages\</NuGetPackageFolders>
+    <NuGetPackageFolders Condition=" '$(NuGetPackageFolders)' == '' ">C:\Users\v-eridah\.nuget\packages\</NuGetPackageFolders>
```
Comment (carlosscastro) [inline CSharp/Library/Microsoft.Bot.Connector.NetCore/Microsoft.Bot.Connector.NetCore.nuget.props]: remove hardcoded paths

File: CSharp/Library/Microsoft.Bot.Connector.NetCore/Microsoft.Bot.Connector.NetCore.nuget.props [hunk 2]
```diff
@@ -16,6 +16,6 @@
     <Import Project="$(NuGetPackageRoot)sourcelink.create.commandline\2.8.3\build\SourceLink.Create.CommandLine.props" Condition="Exists('$(NuGetPackageRoot)sourcelink.create.commandline\2.8.3\build\SourceLink.Create.CommandLine.props')" />
   </ImportGroup>
   <PropertyGroup Condition=" '$(ExcludeRestorePackageImports)' != 'true' ">
-    <PkgNewtonsoft_Json Condition=" '$(PkgNewtonsoft_Json)' == '' ">$(UserProfile)\.nuget\packages\newtonsoft.json\9.0.1</PkgNewtonsoft_Json>
+    <PkgNewtonsoft_Json Condition=" '$(PkgNewtonsoft_Json)' == '' ">C:\Users\v-eridah\.nuget\packages\newtonsoft.json\9.0.1</PkgNewtonsoft_Json>
```
Comment (carlosscastro) [inline CSharp/Library/Microsoft.Bot.Connector.NetCore/Microsoft.Bot.Connector.NetCore.nuget.props]: undo

File: CSharp/Library/Microsoft.Bot.Connector.NetFramework/BotAuthentication.cs [hunk 3]
```diff
@@ -77,7 +77,12 @@ public override async Task OnActionExecutingAsync(HttpActionContext actionContex
             await base.OnActionExecutingAsync(actionContext, cancellationToken);
         }
 
-        private IList<Activity> GetActivities(HttpActionContext actionContext)
+        protected async Task BaseOnActionExecutingAsync(HttpActionContext actionContext, CancellationToken cancellationToken)
```
Comment (carlosscastro) [inline CSharp/Library/Microsoft.Bot.Connector.NetFramework/BotAuthentication.cs]: why do we need this? is base.OnActionExecutingAsync private?

File: CSharp/Library/Microsoft.Bot.Connector.NetFramework/SkillBotAuthentication.cs [hunk 4]
```diff
@@ -0,0 +1,69 @@
+﻿using System;
```
Comment (carlosscastro) [inline CSharp/Library/Microsoft.Bot.Connector.NetFramework/SkillBotAuthentication.cs]: copyright header

File: CSharp/Library/Microsoft.Bot.Connector.NetFramework/BotAuthentication.cs (hunk 3 above)
Comment (EricDahlvang) [reply CSharp/Library/Microsoft.Bot.Connector.NetFramework/BotAuthentication.cs]: Need to call ActionFilterAttribute.OnActionExecutingAsync from SkillBotAuthentication.  Is there a better way to skip the middle inheriting class method (c# doesn't allow base.base.method()) ?  SkillBotAuthentication -> BotAuthentication -> ActionFilterAttribute

Comment (EricDahlvang) [reply CSharp/Library/Microsoft.Bot.Connector.NetFramework/BotAuthentication.cs]: Changed BaseOnActionExecutingAsync to ContinueOnActionExecutingAsync

**Pull request merged**